optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/config/ProblemBenchmarksConfig.java

```diff
@@ -106,6 +108,37 @@ public void setSingleStatisticTypeList(List<SingleStatisticType> singleStatistic
         this.singleStatisticTypeList = singleStatisticTypeList;
     }
 
+    // ************************************************************************
+    // Complex methods
+    // ************************************************************************
+
+    /**
+     * Return the problem statistic type list, or a list containing default metrics if problemStatisticEnabled
+     * is not false. If problemStatisticEnabled is false, an empty list is returned.
+     *
+     * @return never null
+     */
+    public List<ProblemStatisticType> determineProblemStatisticTypeList() {
```

Comment:
I'm thinking... `config` is public API. Do we want these methods there?
Do we have a way of hiding them?

Reply:
It matches what we do for SolverConfig: https://github.com/kiegroup/optaplanner/blob/8ef196252291d00cdf6a033ea3ade9927a919fbf/core/optaplanner-core-impl/src/main/java/org/optaplanner/core/config/solver/SolverConfig.java#L585-L601

Reply:
Fair enough.